services/cloudaccounts/aws_org/aws_org.go

```diff
@@ -0,0 +1,215 @@
+	ManagementAccountStackId      string                                 `json:"managementAccountStackId"`
+	ManagementAccountStackRegion  string                                 `json:"managementAccountStackRegion"`
+	OnboardingConfiguration       AwsOrganizationOnboardingConfiguration `json:"onboardingConfiguration"`
+	UserId                        int                                    `json:"userId"`
+	EnableStackModify             bool                                   `json:"enableStackModify"`
+	StackSetArn                   string                                 `json:"stackSetArn"`
+	OrganizationName              string                                 `json:"organizationName"`
+	UpdateTime                    time.Time                              `json:"updateTime"`
+	CreationTime                  time.Time                              `json:"creationTime"`
+	StackSetRegions               map[string]struct{}                    `json:"stackSetRegions"`
+	StackSetOrganizationalUnitIds map[string]struct{}                    `json:"stackSetOrganizationalUnitIds"`
+}
+
+type OnboardingConfigurationOptions struct {
+	AwsAccountId string `json:"awsAccountId"`
+}
+
+func (service *Service) Create(body OnboardingRequest) (*OrganizationManagementViewModel, *http.Response, error) {
+	v := new(OrganizationManagementViewModel)
+	resp, err := service.Client.NewRequestDo("POST", cloudaccounts.RESTfulServicePathAwsOrgMgmt, nil, body, v)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return v, resp, nil
+}
+
+func (service *Service) UpdateStackSetArn(id string, body UpdateStackSetArnRequest) (*http.Response, error) {
+	if id == "" {
+		return nil, fmt.Errorf("id parameter must be passed")
+	}
+
+	relativeURL := fmt.Sprintf("%s/%s/%s", cloudaccounts.RESTfulServicePathAwsOrgMgmt, id, cloudaccounts.RESTfulServicePathAwsOrgMgmtStacksetArn)
+	resp, err := service.Client.NewRequestDo("PUT", relativeURL, nil, body, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	return resp, nil
+}
+
+func (service *Service) UpdateConfiguration(id string, body UpdateConfigurationRequest) (*http.Response, error) {
+	if id == "" {
+		return nil, fmt.Errorf("id parameter must be passed")
+	}
+
+	relativeURL := fmt.Sprintf("%s/%s/%s", cloudaccounts.RESTfulServicePathAwsOrgMgmt, id, cloudaccounts.RESTfulServicePathAwsOrgMgmtConfiguration)
+	resp, err := service.Client.NewRequestDo("PUT", relativeURL, nil, body, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	return resp, nil
+}
+
+func (service *Service) Delete(id string) (*http.Response, error) {
+	relativeURL := fmt.Sprintf("%s/%s", cloudaccounts.RESTfulServicePathAwsOrgMgmt, id)
+	resp, err := service.Client.NewRequestDo("DELETE", relativeURL, nil, nil, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	return resp, nil
+}
+
+func (service *Service) Get(id string) (*OrganizationManagementViewModel, *http.Response, error) {
```

Comment:
Would it be better to call this function GetByXXXId, so that in the future, if we will have more GetByXXId it will be easier to identify them? In addition, it will make the type of the ID self-explanatory